Gemfile.lock

```diff
@@ -283,9 +286,11 @@ DEPENDENCIES
   rubocop (~> 0.36.0)
   ruby-saml (~> 1.3.0)
   sdoc (~> 0.4.0)
+  shoulda-matchers
```

Comment:
sorry i didn't see your edited comment, i put a binding;pry on this line https://github.com/thoughtbot/shoulda-matchers/blob/547500730dab994b4e06ce97cc521d54bb53f579/lib/shoulda/matchers/active_model/validator.rb#L92
and it's reaching it for my test. it looks like it is testing behavior and not just verifying the macro